docs/docs/configuration/queryable-projections.mdx

```diff
@@ -76,3 +74,48 @@ public static partial class CarMapper
     private static partial CarDto Map(Car car);
 }
 ```
+
+## User-implemented mapping methods
+
+Mapperly tries to inline user-implemented mapping methods.
+For this to work, user-implemented mapping methods need satisfy certain limitations:
+
+- Only expression-bodied methods can be inlined.
+- The body needs to follow the expression tree [limitations](https://learn.microsoft.com/en-us/dotnet/csharp/advanced-topics/expression-trees/#limitations).
+- Nested MethodGroups cannot be inlined.
+
+```csharp
+[Mapper]
+public static partial class CarMapper
+{
+    public static partial IQueryable<CarDto> ProjectToDto(this IQueryable<Car> q);
+
+    // highlight-start
+    private static string MapCarBrandName(CarBrand brand)
+      => band.Name;
+    // highlight-end
+}
+```
+
+If Mapperly is unable to inline a user-implemented method, `RMG068` is reported.
+Non-inlined method invocations can cause the query to be materialized and more data than necessary could be loaded.
```

Comment:
Not sure if this sentence sounds correct? Maybe change to

> Non-inlined method invocations can lead to more data being loaded than necessary.

Reply:
Improved